lib/mix/tasks/git_ops.release.ex

```diff
@@ -57,15 +57,20 @@ defmodule Mix.Tasks.GitOps.Release do
   @doc false
   def run(args) do
     changelog_file = GitOps.Config.changelog_file()
-    path = Path.expand(changelog_file)
-    mix_project_module = GitOps.Config.mix_project()
+    changelog_path = Path.expand(changelog_file)
 
-    unless mix_project_module do
-      raise "mix_project must be configured in order to use git_ops. Please see the configuration in the README.md for an example."
+    unless File.exists?(changelog_path) do
+      raise "\nFile: #{changelog_path} did not exist. Please use the `--initial` command to initialize."
     end
 
-    mix_project = mix_project_module.project()
-    repo = GitOps.Git.init!()
+    mix_project_module = GitOps.Config.mix_project()
+
+    mix_project =
+      if mix_project_module do
+        mix_project_module.project()
+      else
+        raise "mix_project must be configured in order to use git_ops. Please see the configuration in the README.md for an example."
```

Comment:
We should do this in the config module itself, just for organizations sake.

Reply:
And also, since it can't be an environment variable (only compile time application env), we could potentially just raise in a `before_compile` step to make it cleaner.